dll/win32/shell32/wine/shellord.c

```diff
@@ -1391,11 +1391,10 @@ BOOL WINAPI SHRunControlPanel (_In_opt_ LPCWSTR commandLine, _In_opt_ HWND paren
 #ifdef __REACTOS__
     //TODO: Run in-process when possible, (HKLM\Software\Microsoft\Windows\CurrentVersion\Explorer\ControlPanel\InProcCPLs), and possibly some extra rules,
     //TODO: 'If the specified Control Panel item is already running, SHRunControlPanel attempts to switch to that instance rather than opening a new instance.'
-    
+    //TODO: This function is not supported as of Windows Vista and as of Windows Vista, this function always returns FALSE (https://docs.microsoft.com/en-us/windows/win32/api/shlobj/nf-shlobj-shruncontrolpanel )
+    //but we need to keep it "live" even when reactOS is compliled as NT6+ in order to keep control panel elements launch commands.
```

Comment:
I do doubt we would actually want to break NT6 compat...

Reply:
OK, but no need to break it right ? I doubt that any app explicitely rely on the fact that this API fails in VIsta+

Reply:
(For what it's worth, I disagree.)